File: languages/python/bloock/entity/identity_v2/attribute.py
class Attribute:
    """
    Represents an attribute with an identifier and a corresponding value.
    """
    def __init__(
            self, id: str, value
    ) -> None:
        """
        Constructs an Attribute object with the specified parameters.
        :type value: object
        :type id: object
        :rtype: object
        """
        self.id = id
        self.value = value

File: languages/js/src/entity/identity_v2/attribute.ts
/**
 * Represents an attribute with an identifier and a corresponding value.
 */
export class Attribute<T> {
  id: string;
  value: T;

  /**
   * Constructs a Attribute object with the specified parameters.
   * @param id 
   * @param value 
   */
  constructor(id: string, value: T) {
    this.id = id;
    this.value = value;
  }
}
